pkg/platform/microservice/service.go

```diff
@@ -78,23 +78,23 @@ func (s *service) Create(responseWriter http.ResponseWriter, request *http.Reque
 		return
 	}
 
-	tenant := k8s.Tenant{
+	customer := k8s.Tenant{
 		ID:   applicationInfo.Tenant.ID,
 		Name: applicationInfo.Tenant.Name,
 	}
 
-	allowed := s.k8sDolittleRepo.CanModifyApplicationWithResponse(responseWriter, tenant.ID, applicationID, userID)
+	allowed := s.k8sDolittleRepo.CanModifyApplicationWithResponse(responseWriter, customer.ID, applicationID, userID)
 	if !allowed {
 		return
 	}
 
-	if !s.gitRepo.IsAutomationEnabled(tenant.ID, applicationID, environment) {
+	if !s.gitRepo.IsAutomationEnabled(customer.ID, applicationID, environment) {
 		utils.RespondWithError(
 			responseWriter,
 			http.StatusBadRequest,
 			fmt.Sprintf(
 				"Tenant %s with application %s in environment %s does not allow changes via Studio",
```

Comment:
```suggestion
				"Customer %s with application %s in environment %s does not allow changes via Studio",
```

Should we also change it to "Customer" in here? We've really muddied the waters with different types of tenants 😂

Reply:
I vote for gradually changing the "tenants of dolittle" -> "customers". As Joel said, it's a bit tricky to always have to specify which kind of tenant we're talking about in conversations and in the code. I think it would be too big of a job to fix all references right away, but changing as we come across it is fine by me.

Reply:
I agree.  Then I'll commit the suggestion